### ISSUE TITLE
Python 3 invalid syntax

### ISSUE THREAD

**Issue by typoman** [jackjennings/Mechanic#23]
```
Installing 'Mechanic' report:
  Traceback (most recent call last):
      File "_startup.py", line 5, in <module>
      File "/Users/xx/Library/Application Support/RoboFont/plugins/Mechanic.roboFontExt/lib/mechanic/observers/update.py", line 3, in <module>
      File "/Users/xx/Library/Application Support/RoboFont/plugins/Mechanic.roboFontExt/lib/mechanic/threaded.py", line 68
        print "\n".join(errorMessage)
                 ^
    SyntaxError: invalid syntax

```

**Comment by miguelsousa** [comment]
That's an easy fix.
```python
print("\n".join(errorMessage))
```

**Comment by jackjennings** [comment]
Fixed now in #20 — thanks for reporting!
